v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "    inputs=[model_data],\n",
+    "    compute_target=compute_target,\n",
+    "    runconfig=conda_run_config,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Model name: {model_name_str}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.3.3 Build pipeline\n",
+    "\n",
+    "Next, we build the pipeline and kick off a run which will select the best TCN model and register it in the workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline = Pipeline(\n",
+    "    description=\"training_pipeline_tcn\",\n",
+    "    workspace=ws,\n",
+    "    steps=[automl_step, register_model_step],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Is the TCN model trained? {IS_TCN_MODEL_TRAINED}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if not IS_TCN_MODEL_TRAINED:\n",
+    "    print(\"Training new AutoML model ...\\n---\")\n",
+    "    training_pipelin_run = experiment.submit(training_pipeline)\n",
+    "    training_pipelin_run.wait_for_completion(show_output=False)\n",
+    "    IS_TCN_MODEL_TRAINED = True\n",
+    "else:\n",
+    "    from azureml.train.automl.run import AutoMLRun\n",
+    "    from azureml.pipeline.core.run import PipelineRun\n",
+    "\n",
+    "    PIPELINE_RUN_ID = (\n",
+    "        \"<Replace with the PipelineRunId used for training the best TCN model>\"\n",
+    "    )\n",
+    "    training_pipelin_run = PipelineRun(experiment=experiment, run_id=PIPELINE_RUN_ID)\n",
+    "    print(f\"Using previously trained model. Pipeline run ID: {PIPELINE_RUN_ID}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Train the baseline model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use Naive model as our baseline. To train it, we kick off another automl experiment with the following settings. Please note that we disabled DNN (`enable_dnn=False`), added the Naive model to the allowed models list, and set the number of iterations to 1 since we only interested in training one specific model. To reduce the training time, we set the number of cross validations to 2. Read the following [link](https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-auto-train-forecast-v1#training-and-validation-data) for more information on this topic. Note that we did not use the cross validations settings for the TCN model training because such models typically require validation set that are substantially longer than a small multiple of a forecast horizon.\n",
```

Comment:
Should we add link for the allowed models? Maybe [https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.forecastingmodels?view=azure-python#fields](url) ? The customer might want to use a different baseline based on the data, it would be easier to add the required model quickly for baseline.

Reply:
There is already a discussion with references on how to change the baseline model with appropriate links to documentation. The URL you mentioned lists only the forecasting models and this is not a complete list of models we support. We also support regression models for the forecasting tasks.